<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NovellVpnWidget</class>
 <widget class="QWidget" name="NovellVpnWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novell VPN</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Gateway:</string>
        </property>
        <property name="buddy">
         <cstring>leGateway</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KLineEdit" name="leGateway"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Gate&amp;way Type:</string>
        </property>
        <property name="buddy">
         <cstring>cmbGwType</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbGwType">
        <item>
         <property name="text">
          <string>Nortel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Standard Gateway</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Authentication</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;Type:</string>
        </property>
        <property name="buddy">
         <cstring>cmbAuthType</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbAuthType">
        <item>
         <property name="text">
          <string>XAUTH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X.509</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QStackedWidget" name="authTypeStack">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&amp;Username:</string>
            </property>
            <property name="buddy">
             <cstring>leUserName</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KLineEdit" name="leUserName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&amp;Group Name:</string>
            </property>
            <property name="buddy">
             <cstring>leGroupName</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KLineEdit" name="leGroupName"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>U&amp;ser Password:</string>
            </property>
            <property name="buddy">
             <cstring>leUserPass</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="KLineEdit" name="leUserPass">
            <property name="passwordMode">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>G&amp;roup Password:</string>
            </property>
            <property name="buddy">
             <cstring>leGroupPass</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="KLineEdit" name="leGroupPass">
            <property name="passwordMode">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Certificate &amp;file:</string>
            </property>
            <property name="buddy">
             <cstring>x509Cert</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KUrlRequester" name="x509Cert"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Certificate &amp;password:</string>
            </property>
            <property name="buddy">
             <cstring>x509CertPass</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KLineEdit" name="x509CertPass">
            <property name="passwordMode">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cbShowPasswords">
        <property name="text">
         <string>&amp;Show Passwords</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KPushButton" name="btnAdvanced">
        <property name="text">
         <string>Ad&amp;vanced...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cmbAuthType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>authTypeStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
